Source/AssistantV2/Models/SearchResultHighlight.swift

```diff
@@ -1,5 +1,5 @@
 /**
- * (C) Copyright IBM Corp. 2019, 2020.
+ * (C) Copyright IBM Corp. 2019.
```

Comment:
This looks odd -- should be investigated.  Was the 2020 date in the original copyright wrong?